<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelineWidget</class>
 <widget class="QWidget" name="TimelineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="logInfoLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="reloadTb">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/sync_black.svg</normaloff>:/viewer/sync_black.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="pathFilterLe"/>
     </item>
     <item>
      <widget class="QToolButton" name="taskOnlyTb">
       <property name="text">
        <string>Tasks only</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="startTb">
       <property name="toolTip">
        <string>Extend period to the start</string>
       </property>
       <property name="text">
        <string>start</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/arrow_to_start.svg</normaloff>:/viewer/arrow_to_start.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="fromTimeEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="toTimeEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="endTb">
       <property name="toolTip">
        <string>Extend period to the end</string>
       </property>
       <property name="text">
        <string>end</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/arrow_to_end.svg</normaloff>:/viewer/arrow_to_end.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="wholePeriodTb">
       <property name="statusTip">
        <string>Show full period</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/arrow_all.svg</normaloff>:/viewer/arrow_all.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="todayTb">
       <property name="text">
        <string>Today</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="yesterdayTb">
       <property name="text">
        <string>Yesterday</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MessageLabel" name="messageLabel" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="viewHolder" native="true">
     <layout class="QVBoxLayout" name="viewHolderLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
